torch/_subclasses/fake_tensor.py

```diff
@@ -803,10 +803,18 @@ def __torch_dispatch__(self, func, types, args=(), kwargs=None):
 
         converter = self.fake_tensor_converter
 
-        # If this is a lift, the input tensor is guaranteed to be a
-        # constant, so we keep a copy of the original argument along so
-        # we can query it if we're asked to item() it at some later point
-        if func in self.lift_fns:
+        # To constant propagate through these functions:
+        # 1, If this is a lift, the input tensor is guaranteed to be a
+        #    constant, so we keep a copy of the original argument along so
+        #    we can query it if we're asked to item() it at some later point
+        # 2, Some functions that allow Python numbers to bind to Tensors, e.g, torch.div
+        if func in self.lift_fns or (
+            torch._C._should_allow_numbers_as_tensors(
+                func.name().split("::")[-1].split(".")[0]
+            )
```

Comment:
there is already code:
```
torch._C._should_allow_numbers_as_tensors(
                func.name().split("::")[-1].split(".")[0]
            )
```

Elsewhere in this file. It is not clear what the func name is, nor what it expects to turn it into for `_should_allow_numbers_as_tensors`

`_should_allow_numbers_as_tensors` is not documented either.

Please refactor this into a util (since it is used in > 1 places now) and add a comment indicating what you expect the func/func name to be, and what you expect to pass to `_should_allow_numbers_as_tensors`